lamby/lamby.py

```diff
@@ -44,11 +50,112 @@ def uninit():
 @click.argument('files', nargs=-1)
 @click.option('-m', '--message')
 def commit(files, message):
-    click.echo('Commiting following files:')
+    lamby_dir = './.lamby'
+    if not os.path.isdir(lamby_dir):
+        click.echo('Lamby project has not been initialized in ' + os.getcwd())
+        sys.exit(1)
+
+    if len(files) == 0:
+        files = search_file_type('.', 'onnx')
```

Comment:
Probably should check if files is empty

Reply:
I believe if no files are specified, click will pass in an empty array, but doesn't hurt to do a NoneType check.

Reply:
I meant if search_file_type returns an empty array. Maybe show some error message that says no onnx file in dir?